### ISSUE TITLE
A "sorry, i did not understand" responder?

### ISSUE THREAD

**Issue by mpadge** [ropensci-org/buffy#41]
@xuanxu We recently received [an incorrectly formatted command](https://github.com/ropensci/software-review/issues/475#issuecomment-954880050) which made us realise the importance of some kind of generic, "sorry, i did not understand that command" response. Would that be possible to implement? It took us four days to realise that something had gone wrong there, but a responder like that would have enabled immediate diagnosis.

**Comment by xuanxu** [comment]
Yes, this is something I have in my backlog for Buffy, I've just created [an issue](https://github.com/openjournals/buffy/issues/52) upstream.
